src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.repositories;
+
+import org.springframework.stereotype.Repository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends Repository {
+
+	public Book findBookByAuthor(String author);
```

Comment:
this method should return an Optional<Book> to avoid nullchecks whenever possible :)